onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -48,9 +47,9 @@ def get_inputs(args: argparse.Namespace, ort_model_inputs_len: int):
     init_inputs, iter_inputs = None, None
 
     # For past_present_share_buffer:
-    # Set max_seq_len to 2048 for Hugging Face model since that is the default value
+    # Set max_seq_len to 4096 for Hugging Face model since that is the default value
     # Set max_seq_len to 2048 for Microsoft model since that is the max value currently supported
-    max_seq_len = 2048
+    max_seq_len = 2048 if args.benchmark_type == "ort-msft" else 4096
```

Comment:
What will happen if user set larger sequence length for benchmarking? say --sequence-lengths 8192

Reply:
According to [this documentation](https://huggingface.co/docs/transformers/model_doc/llama2#transformers.LlamaConfig.max_position_embeddings):
- LLaMA supports up to 2048
- LLaMA-2 supports up to 4096
- CodeLLaMA supports up to 16384

I'll change this to be an optional user argument in case users try the other variants with larger sequence lengths.

Reply:
I think no need to use an extra argument, use 4096 or the largest of sequence_lengths list given by user, whichever is larger

Reply:
Fixed